README.rst

```diff
@@ -0,0 +1,358 @@
+=====================
```

Comment:
We tend to rely on markdown instead of RST. I'm happy to update after the first commit manually though.